internal/api/v1/tenant.go

```diff
@@ -0,0 +1,69 @@
+package v1
+
+import (
+	"net/http"
+
+	"github.com/flexprice/flexprice/internal/api/dto"
+	"github.com/flexprice/flexprice/internal/logger"
+	"github.com/flexprice/flexprice/internal/service"
+	"github.com/gin-gonic/gin"
+)
+
+type TenantHandler struct {
+	service service.TenantService
+	log     *logger.Logger
+}
+
+func NewTenantHandler(service service.TenantService, log *logger.Logger) *TenantHandler {
+	return &TenantHandler{service: service, log: log}
+}
+
+// @Summary Create a new tenant
+// @Description Create a new tenant
+// @Tags Tenant
+// @Accept json
+// @Produce json
+// @Security BearerAuth
+// @Param request body dto.CreateTenantRequest true "Create tenant request"
+// @Success 201 {object} dto.TenantResponse
+// @Failure 400 {object} environmentErrorResponse
+// @Failure 500 {object} environmentErrorResponse
```

Comment:
where is this environmentErrorResponse defined?

Reply:
it should be Error Response : fixed